lib/src/clojure_lsp/refactor/transform.clj

```diff
@@ -37,6 +37,19 @@
                 (dissoc :loc))))
         zip-edits))
 
+(defn- unify-to-one-language
```

Comment:
Should this go to queries.clj or do you think it's a overhead?

Reply:
I just moved this function up in the ns, so I don't know. Probably outside scope tho of this tho.